kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/GetColumnsRequest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.client.api.v1.dto;
+
+import java.util.Objects;
+
+public class GetColumnsRequest {
+  private String catalogName;
+  private String schemaName;
+  private String tableName;
+  private String columnName;
+
+  public GetColumnsRequest() {}
+
+  public GetColumnsRequest(
+      String catalogName, String schemaName, String tableName, String columnName) {
```

Comment:
```
String catalogName,
String schemaName,
String tableName,
String columnName
```

Reply:
I try to modify it, but dev/reformat will revert my change.